src/core/redux/reducers/resetRequestStates.js

```diff
@@ -4,16 +4,19 @@
 
 import { Map } from 'immutable';
 import { RequestStates } from 'redux-reqseq';
+import { LangUtils } from 'lattice-utils';
 
 import { ERROR, REQUEST_STATE } from '../constants';
 import type { ResetRequestStatesAction } from '../actions';
 
+const { isNonEmptyString } = LangUtils;
+
 export default function reducer(state :Map, action :ResetRequestStatesAction) {
 
   const { actions = [] } = action;
   let newState = state;
   actions.forEach((requestStateAction :string) => {
-    if (state.hasIn([requestStateAction])) {
+    if (isNonEmptyString(requestStateAction) && state.hasIn([requestStateAction])) {
```

Comment:
Curious why you're adding this check? Did you run into an issue?

Reply:
ugh - I really should have reverted all of this. Before I broke out the singular `resetRequestState` I had altered this to check whether the `requestStaateAction` was a non-empty string or a non-empty array path. I did this to handle reducers where the request state was stored one step further in [ACTION, EKID].